projects/addon-editor/components/editor-socket/editor-socket.component.ts

```diff
@@ -48,6 +60,9 @@ export class TuiEditorSocketComponent {
         @Optional()
         @Inject(TUI_SANITIZER)
         private readonly tuiSanitizer: Sanitizer | null,
+        @Inject(PreviewDialogService)
+        private readonly previewDialogService: PreviewDialogService,
```

Comment:
Should current users have `TuiPreviewModule` and addon-preview package? Can we keep it optional ?

Reply:
Not sure, because we are importing modules in editor module

Reply:
so what about package addon-preview? shouldn't we add it to addon-editor dependencies?